controllers/front/callbackfail.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Altapay module for Prestashop
+ *
+ * Copyright © 2020 Altapay. All rights reserved.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+require_once _PS_MODULE_DIR_ . '/altapay/lib/altapay/altapay-php-sdk/lib/AltapayCallbackHandler.class.php';
+require_once _PS_MODULE_DIR_ . '/altapay/helpers.php';
+
+class AltapayCallbackfailModuleFrontController extends ModuleFrontController
+{
+    /**
+     * Method to add external assets
+     */
+    public function setMedia()
+    {
+        parent::setMedia();
+        $this->addCSS(($this->module->getPathUri()) . 'css/altapay.css', 'all');
+    }
+
+    /**
+     * Method to follow when callback fail is being triggered
+     * @throws Exception
+     */
+    public function postProcess()
+    {
+        $xml = Tools::getValue('xml');
+        $callbackHandler = new AltapayCallbackHandler();
+        $response = $callbackHandler->parseXmlResponse($xml);
+
+        $shopOrderId = $response->getPrimaryPayment()->getShopOrderId();
+
+        // load the cart
+        $cart = getCartFromUniqueId($shopOrderId);
+        if (!Validate::isLoadedObject($cart)) {
+            die('Could not load cart - exiting');
```

Comment:
Maybe avoid function aliases, at least consistently.
```suggestion
            exit('Could not load cart - exiting');
```